### PR TITLE
paragraph and figure showing the interconnections between the sections

### PR DESCRIPTION
**Pull request by pchampin**
the proposed text and figure address issue #203

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-star/pull/206.html" title="Last updated on Sep 17, 2021, 11:55 AM UTC (f701c0c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-star/206/1ccc91e...f701c0c.html" title="Last updated on Sep 17, 2021, 11:55 AM UTC (f701c0c)">Diff</a>

### CODE REVIEW

**Review by afs — APPROVED** [review]
Ideally, a lines from the centre to the surrounding boxes would be nice - it breaks up the table like appearance. But it is yet more work.

File: cg-spec/editors_draft.html [hunk 1]
```diff
@@ -352,6 +362,51 @@ <h2>Overview</h2>
 
     </section>
 
+    <section id="structure">
+      <h2>Structure of this document</h2>
+      <p>This specification covers many aspect of RDF-star, and not all sections will be of interest to all readers.</p>
+      <p><a href="#concepts">Section 2</a> first describes the concepts and abstract syntax of RDF-star, to which all following sections refer.
+      The next three sections focus on how users will interact with the abstract syntax, describing the <a href="#concrete-syntaxes">concrete syntaxes</a>, the <a href="#sparql-star">query language SPARQL-star</a> and its <a href="#sparql-star-update">counterpart for updates</a>. <a href="#rdf-star-semantics">Section 6</a> then describes RDF-star's formal semantics, which lays the foundation for reasoning with RDF-star data. Finally, <a href="#rdf-star-vocabulary">section 7</a> defines an RDF vocabulary capturing the main concepts of RDF-star.</p>
```
Comment (hartig) [inline cg-spec/editors_draft.html:369]:
```suggestion
      <p>This specification covers many aspects of RDF-star, and not all sections will be of interest to all readers.</p>
      <p><a href="#concepts">Section 2</a> first defines the concepts and the abstract syntax of RDF-star, to which all following sections refer.
      The next three sections focus on how users will interact with the abstract syntax, defining the <a href="#concrete-syntaxes">concrete syntaxes</a>, the <a href="#sparql-star">query language SPARQL-star</a> and its <a href="#sparql-star-update">counterpart for updates</a>. <a href="#rdf-star-semantics">Section 6</a> then defines RDF-star's formal semantics, which lays the foundation for reasoning with RDF-star data. Finally, <a href="#rdf-star-vocabulary">Section 7</a> defines an RDF vocabulary capturing the main concepts of RDF-star.</p>
```

**Review by hartig — APPROVED** [review]
Nice! I have just a few suggestions for the paragraph.

**Review by gkellogg — APPROVED** [review]

**Comment by pchampin** [comment]
> Ideally, a lines from the centre to the surrounding boxes would be nice - it breaks up the table like appearance. But it is yet more work.

Yes that was my initial plan. I tried SVG, but it was too much work :-/

**Comment by TallTed** [comment]
FWIW, there are graphic editors that can export to reasonable SVG which have been used in other groups, such as the VCWG and CCG. [Google Drawings](https://docs.google.com/drawings) is one such.

**Comment by pchampin** [comment]
@TallTed the thing is:

* I want the SVG to be embedded in the HTML (not transcluded with ``), in order to have links
* Ideally, I want to let RESPEC generate those links for me (filling automatically the section titles, so that they don't run out-of-sync)

Granted, none of this is absolutely blocking, but I considered that the benefit/effort ratio was not worth it. If there is a consensus that the figure must be improved, I'll do it.

**Comment by rat10** [comment]
Regarding the central bubble: if no good title can be found  to make it as prominent as the other bubbles then just add a blank line before and after the text (or CSS padding top and bottom). That should give it the right level of prominence visually.

**Comment by pchampin** [comment]
merged, as resolved during today's meeting:
https://w3c.github.io/rdf-star/Minutes/2021-10-01.html#r01

**Pull request merged**